src/librustc_driver/lib.rs

```diff
@@ -346,20 +346,22 @@ pub fn run_compiler(
 
             queries.global_ctxt()?;
 
+            // Drop AST after creating GlobalCtxt to free memory
+            let _timer = sess.prof.generic_activity("drop_ast");
+            mem::drop(queries.expansion()?.take());
```

Comment:
This needs a block around it, the `_timer` is supposed to be dropped after the next statement, in order for `drop_ast` to track only that, and not the rest of this large block. (it's not a big deal, but I was baffled as to why `drop_ast` had so much variance, or any children at all, in the profiling data)

Reply:
Sorry about that! Should be fixed by https://github.com/rust-lang/rust/pull/75575